snafu/uperf_wrapper/trigger_uperf.py

```diff
@@ -37,6 +37,13 @@ def __init__(self, args):
         self.num_pairs = args.num_pairs
         self.multus_client = args.multus_client
         self.networkpolicy = args.networkpolicy
+        self.nodes_in_iter = args.nodes_in_iter
```

Comment:
I'd prefer to make this variable something more descriptive as nodes_in_iteration.

Reply:
I thought `iter` was a sufficient and well known abbreviation, its already a long variable.

Reply:
Yeh, I'm not sure about the well known abbreviation part, maybe we can ask someone else to weight in. Not married to any name.